cafe.rb

```diff
@@ -19,15 +19,15 @@ def take_order(menus)
     puts "(#{i})#{menu[:name]}: #{menu[:price]}円"
   end
   print '>'
-  order_number = gets.to_i
-  puts "#{menus[order_number - 1][:name]}(#{menus[order_number - 1][:price]}円)ですね。"
+  order_number = gets.to_i - 1
+  puts "#{menus[order_number][:name]}(#{menus[order_number][:price]}円)ですね。"
   order_number
 end
 
 puts 'bugカフェへようこそ！ご注文は？ 番号でどうぞ'
-order1 = take_order(DRINKS)
+order_drink_number = take_order(DRINKS)
```

Comment:
orderで始めると「注文する」という動詞に見える＝メソッドに見える可能性があるので、

```suggestion
drink_number = take_order(DRINKS)
```

だけでもいいかなと思いました。